cmd/analyzer/analyzer.go

```diff
@@ -35,10 +55,13 @@ type cliFlags struct {
 	memoryRecordsTtl time.Duration
 }
 
-func parseFlags() cliFlags {
+func parseFlags(version string) cliFlags {
 	cli := cliFlags{}
 	fs := flag.NewFlagSet("analyzer", flag.ExitOnError)
 
+	fs.BoolVar(&cli.json, "json", false, "Print application info as json")
+	fs.BoolVar(&cli.json, "j", false, "Print application info as json (shorthand)")
```

Comment:
Never noticed this, surprised `flag` package has no built-in shorthands 😮

Reply:
Dropping the full flag `-json`, as mist runtime has no use for it anyway.